app/controllers/admin_controller.rb

```diff
@@ -139,7 +139,7 @@ def mark_spam
   def mark_comment_spam
     @comment = Comment.find params[:id]
     if current_user && (current_user.role == 'moderator' || current_user.role == 'admin')
-      if @comment.status == 1
+      if @comment.status == 1 || @comment.status == 4
```

Comment:
Change in mark_comment_spam so that moderated comment can be marked spam.

Reply:
Got it. Do I have to create a new pull request for that?